### DISCUSSION TITLE
Comparison and Strengths of DeepTB, DeepH, and DeePMD

### DISCUSSION THREAD

**Discussion by lnnbig** [deepmodeling/deepmd-kit#2754] [General]
When focusing on atomic structure only, as opposed to electronic structure, what specific advantages does DeepTB bring to the table when compared to DeePMD or DeepH? Could someone explain the strengths in each of these three codes? Thanks!

**Comment by jameswind** [comment]
When focusing on atomic structure only, DeePMD should be the way to go, right? 
If the electronic structure method is too expensive, you may consider DeePKS as a bridge. See, e.g., https://pubs.acs.org/doi/10.1021/acs.jpca.2c05000 (open-source in DeepModeling)
As for DeepH and DeePTB, they are methods for learning reduced electronic structure models. The object to be learned is a bit different: DeepH directly learns the overlap matrix generated by the coefficients of atomic orbitals after an SCF calculation. The learned object could be large and non-orthogonal, but the learning process itself should be straightforward; DeePTB targets a more reduced tight-binding Hamiltonian, making the representation and training of the model more challenging, but the resulting Hamiltonian is more compact and there is no requirement for the basis for solving DFT to be atomic (not planewave). However, if you focus on atomic structures, in my view, you may go directly with DeePMD, or DeePMD+DeePKS.
